documentation/writing_plans.md

```diff
@@ -173,6 +173,73 @@ parameterized `Sensitive` type, such as `Sensitive` or `Sensitive[Hash]`. Other
 types, such as `Optional[Sensitive]` or `Hash[String, Sensitive]`, will not be
 automatically masked.
 
+## Using Hiera data in plans
+
+You can use the Puppet `lookup()` function in plans to look up Hiera data. It's useful to think
+of looking up Hiera data in Bolt plans in two different contexts: inside [apply
+blocks](applying_manifest_blocks.md) and outside apply blocks. Inside apply blocks Bolt
```

Comment:
```suggestion
blocks](applying_manifest_blocks.md) and outside apply blocks. Inside apply blocks, Bolt
```

Reply:
Oh my gosh Tom literally had the same comment